en/flight_controller/pixhawk_autopilot_bus.md

```diff
@@ -0,0 +1,25 @@
+# Pixhawk Autopilot Bus & Carriers
```

Comment:
@bkueng I was wondering if you could have a quick look at this page? (below). This adds a page for PAB explaining what it is and offers. The main thing being that it helps users understand that they can plug and play FCs and boards between vendors, allowing flexibility for integrating with companions, but also to choose cheaper or more compact carrier designs. 

It also serves as a place to put compliant carrier boards - these shouldn't really be below a particular FC.

The question for you is whether this should live in the hierarchy 
a) where I have it (in FC section, after the FCs)
b) Higher up in FC section - specifically before FC listings, and after "Pixhawk Series" 
c) Inside FC peripherals
d) As a peer of Flight Controllers and FC peripherals.

Wasn't entirely sure of the right level of visibility.

Reply:
Not quite sure, probably most users will just use the bundled package. Keeping it here makes sense, maybe also one level up (d)).
Longer-term I think it makes sense to have a separate section for FC+mission computer bundled boards as these add more capabilities.

Reply:
@bkueng Thanks very much. That's pretty much my take on it.

> Longer-term I think it makes sense to have a separate section for FC+mission computer bundled boards as these add more capabilities.

I agree, though this is a challenge. For now I just have a section under companion computers: http://docs.px4.io/main/en/companion_computer/#integrated-companion-flight-controller-boards